### DISCUSSION TITLE
Sidebar text is on top of the new search bar

### DISCUSSION THREAD

**Discussion by eliassjogreen** [community/community#38774] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

![example](https://user-images.githubusercontent.com/19556658/201101938-f70eb633-435f-4d97-a8ad-220ffba8fed8.png)

**Comment by ossanna16** [comment]
Hi there @eliassjogreen and welcome to our community! Thank you for reporting this!

**Comment by Akash1134** [comment]
Hi @eliassjogreen, Just wanted to check in if you are still facing this issue with the UI. I am not able to reproduce the same on my end. Are you able to replicate It in a different browser as well? Thanks!

**Reply by eliassjogreen** [reply]
Ah, I investigated a bit and turns out this is an issue with the https://github.com/refined-github/refined-github extenstion
